experimental/packages/opentelemetry-instrumentation-http/src/http.ts

```diff
@@ -25,11 +25,8 @@ import {
   SpanStatus,
   SpanStatusCode,
   trace,
-  Histogram,
-  MeterProvider,
-  MetricAttributes,
-  ValueType,
 } from '@opentelemetry/api';
+import { Histogram, MetricAttributes, ValueType } from '@opentelemetry/api-metrics';
```

Comment:
These types are now part of @opentelemetry/api package, with the PR https://github.com/open-telemetry/opentelemetry-js/pull/3374 landed.

Reply:
Cool! fixed